### ISSUE TITLE
Homepage : should display status and sent date (if sent) for each expenses

### ISSUE THREAD

**Issue by pgayvallet** [publicissapient-france/xndf#41]
Homepage expense listing should show theses informations :
- current status ( draft or sent )
- the sent date if status is sent

Which should allow the user to check if expenses have been sent on time.

**Comment by jeantil** [comment]
Agreed, we need to rethink the UI of the home page for this to fit on a phone browser though ...
